docs/content/app-development.rst

```diff
@@ -0,0 +1,23 @@
+App Development
```

Comment:
"App Development" section is great. I wouldn't put anything besides a table of contents in the main section. Sub-section would contain the actual content